examples/express-ts/package.json

```diff
@@ -14,24 +14,24 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.16.1",
-    "express": "^4.14.1",
-    "express-promise-router": "^1.1.1",
-    "knex": "^0.12.7",
+    "body-parser": "^1.17.2",
+    "express": "^4.15.3",
+    "express-promise-router": "^2.0.0",
+    "knex": "^0.13.0",
     "lodash": "^4.17.4",
-    "morgan": "^1.8.1",
-    "objection": "^0.7.0",
+    "morgan": "^1.8.2",
+    "objection": "../..",
```

Comment:
IMHO, I'd actually add the examples and tests to ci, which would predicate this change. 

I think the people that just clone this folder to start their project should use a release tag, but it might be better to tell people to use a [generator](https://expressjs.com/en/starter/generator.html).
Do you want me to back out this change, though?